docs/schema.md

```diff
@@ -387,6 +387,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>SUSE and openSUSE</code></td>
```

Comment:
This needs to be the record prefix, which from a brief eyeball of https://lists.suse.com/pipermail/sle-security-updates/ should be `SUSE-SU`?

```suggestion
      <td><code>SUSE-SU</code></td>
```

Reply:
changed, with and not in code